python/lsst/pipe/analysis/visitAnalysis.py

```diff
@@ -399,13 +408,13 @@ def readCatalogs(self, dataRefList, dataset, ccdKey, hscRun=None):
                         continue
                 else:
                     # Check for both jointcal_wcs and wcs for compatibility with old datasets
-                    if (not (dataRef.datasetExists("jointcal_wcs") or dataRef.datasetExists("wcs"))
-                        or not dataRef.datasetExists("fcr_md")):
+                    if (not (dataRef.datasetExists("jointcal_wcs") or dataRef.datasetExists("wcs")) or not
+                        dataRef.datasetExists("fcr_md")):
```

Comment:
pep8 E129: indent matches the line below. Most accepted fix is to indent line 412 "dataRef" an extra 4 lines.  Just checking I understand this line: We want to continue if EITHER fcr_md doesn't exist OR fcr_md exists and both jointcal_wcs and wcs don't exist?

Reply:
Yeah.  The point here is that I want both the photometric and astrometric corrections from `meas_mosaic` to exist.  The name of the former has not changed, but that of the latter has, so I check for existence of either the new **jointcal_wcs** or the old **wcs** dataset (for backwards compatibility).  [And note that the function that applies the correction in `meas_mosaic` also does a check for both...for now.]